packages/odyssey-storybook/src/components/odyssey-mui/Button/Button.stories.tsx

```diff
@@ -25,41 +26,135 @@ const storybookMeta: Meta<ButtonProps> = {
   argTypes: {
     isDisabled: {
       control: "boolean",
+      description: "If `true`, the button is disabled",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+        defaultValue: "",
+      },
     },
     isFullWidth: {
       control: "boolean",
+      description:
+        "If `true`, the button will take up the full width available",
+      table: {
+        type: {
+          summary: "boolean",
+        },
+        defaultValue: "",
+      },
     },
     size: {
       options: ["small", "medium", "large"],
       control: { type: "radio" },
+      description: "The size of the button",
+      table: {
+        type: {
+          summary: "small | medium | large",
+        },
+        defaultValue: {
+          summary: "medium",
+        },
+      },
```

Comment:
Is there a reason we're manually filling in the table values? Do they not auto-generate properly?

Reply:
AFAIK, to get the nice type display we need to do it this way.

Without the manually-filled table values:
<img width="1054" alt="Screenshot 2023-06-20 at 9 18 11 PM" src="https://github.com/okta/odyssey/assets/91091570/29d85d95-7f19-41b0-b448-f7948464ae5a">

With the manually-filled table values:
<img width="1036" alt="Screenshot 2023-06-20 at 9 18 50 PM" src="https://github.com/okta/odyssey/assets/91091570/241aafe5-2f79-4efd-b8ef-b42946e1ef6e">

Reply:
We can generate that from the type right? There's gotta be some way.

I used to use enums, so that would've been enumerable: `Object.keys(Size).join(' | ')`. But we can't do that with unions as far as I understand. Is there a way to take the options away and do the `.join` on it?

This is fine though. No further issues.